sitemap.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+      xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<!-- created with Free Online Sitemap Generator www.xml-sitemaps.com -->
+
+
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>1.00</priority>
+</url>
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/content.html</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/story.html</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/privacy.html</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+

```

Comment:
Please see my question in another Pull Request that has this here: https://github.com/nathandh/mywebclass-simulation/pull/39#discussion_r1142870295